docs/adr/0017-python-anlaytics-processing.md

```diff
@@ -0,0 +1,205 @@
+# Python Analytics Processing
+
+## Status: Proposed
+
+## Context
+
+Our team has developed a natural language processing model for the identifying duplicate goals using python. We need to productionize this model and deploy it to our cloud infrastructure on cloud.gov, which has specific technical limitations. We have two design options to consider - containerizing the Python code and deploying it using cloud.govs underlining technology through Cloud Foundry's Docker support, or using Node child processes to wrap execution Python code on one of the workers.
+
+## Alternatives Considered
+
+We have considered several other alternatives, such as using a serverless architecture, deploying the code as a standalone service, and using Kubernetes. However, we have rejected these alternatives for reasons such as complexity, cost, and mainly lack of compatibility with cloud.gov and its technical limitations. There is also the option of using a different cloud provider which would reduce friction, but we have decided to stick with cloud.gov as the boundary would be too large to change at this point. We have also considered using a different programming language, such as R, but we have decided to stick with Python as it is the language that the model was developed in and offers the most flexibility in terms of deployment options.
```

Comment:
Curious about whether or not we think using GovCloud just for this would involve ATO. I presume that it wouldn't. If that's the case, then spinning up a separate server or leveraging a lambda over there entirely is something we eliminated because of the other reasons mentioned? I.E. complexity or boundary changing? 

It feels to me like a much more straightforward way to go about this (although it might be more work up front), at least in part because we want to end up there at the end of the day.